src/view/com/util/Toast.web.tsx

```diff
@@ -78,12 +143,12 @@ const styles = StyleSheet.create({
     bottom: 20,
     // @ts-ignore web only
     width: 'calc(100% - 40px)',
-    maxWidth: 350,
+    maxWidth: 380,
     padding: 20,
     flexDirection: 'row',
-    alignItems: 'center',
-    backgroundColor: '#000c',
+    alignItems: 'flex-start',
```

Comment:
making items flex-start so that the icon can be aligned to the top instead of center when the text takes up multiple lines

Reply:
In the future leave this comment in the code as

```
// NOTE(APiligrim)
// making items flex-start so that the icon can be aligned to the top instead of center when the text takes up multiple lines
```